src/components/FloatingButtons/FloatingButtons.js

```diff
@@ -0,0 +1,107 @@
+// @flow
+/*
+    Pillar Wallet: the personal data locker
+    Copyright (C) 2021 Stiftung Pillar Project
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+
+import * as React from 'react';
+import styled, { useTheme } from 'styled-components/native';
+
+// Components
+import { BaseText } from 'components/Typography';
+
+// Utils
+import { compactFalsy } from 'utils/common';
+import { spacing } from 'utils/variables';
+
+type RenderIcon = React.Node | (({ color: string }) => React.Node);
+
+export type Item = {|
+  title: string,
+  icon: RenderIcon,
```

Comment:
My intent here was to provide flexible yet simple way to provide various types of icons for the buttons: font, images, sgs, etc.

Using `icon` component prop gives the caller most flexibility for providing icons. However, for proper look icons need to have matching color (incl. light/dark theme), hence the option with render prop passing required color.

Please compare use-case in FloatingButtons.stories.js.

Regarding mixing (font) Icon with Svg Icon, this could work. However, there is a downside of only providing font icons or svg icons, without option to use `Image`/`PNG` icons.

Reply:
One could leave the option for other formats by interpreting a string as an icon name, but functions/react nodes as they are now. The available designs all use single-colour icons, so current version means repeating that small render function for every one of them, just in case there's a need for a raster one in the future.

Regarding Storybook, though, I get a runtime error when trying to view them -- commit 8962f191a, the error is 

```
Error: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: number.

Check the render method of `IconSvg`.
```